dm/dumpling/dumpling_test.go

```diff
@@ -124,3 +126,26 @@ func (d *testDumplingSuite) TestDefaultConfig(c *C) {
 	c.Assert(dumpling.dumpConfig.StatementSize, Not(Equals), export.UnspecifiedSize)
 	c.Assert(dumpling.dumpConfig.Rows, Not(Equals), export.UnspecifiedSize)
 }
+
+func TestCallStatus(t *testing.T) {
+	conf := export.DefaultConfig()
+	cfg := &config.SubTaskConfig{}
+	m := &Dumpling{cfg: cfg}
+	ctx := context.Background()
+
```

Comment:
also check `m.Status` before `m.realdumper` is assigned

Reply:
I tried to check before `m.realdumper`, but it always failed, so please give me more advice